spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/DefaultSubscriberFactory.java

```diff
@@ -21,16 +21,22 @@
 import com.google.api.gax.rpc.TransportChannelProvider;
 import com.google.cloud.pubsub.v1.MessageReceiver;
 import com.google.cloud.pubsub.v1.Subscriber;
+import com.google.cloud.pubsub.v1.SubscriptionAdminSettings;
+import com.google.pubsub.v1.PullRequest;
 import com.google.pubsub.v1.SubscriptionName;
 
 import org.springframework.cloud.gcp.core.GcpProjectIdProvider;
 import org.springframework.util.Assert;
+import org.threeten.bp.Duration;
```

Comment:
I think this is wrong import. I guess you thought about `java.time.Duration`.

Anyway this looks like unused one...

Reply:
Unfortunately, the import package is right, but it's not needed here.